packages/story-editor/src/constants/index.js

```diff
@@ -96,3 +96,5 @@ export const OUTLINK_THEME = {
   LIGHT: 'light',
   CUSTOM: 'custom',
 };
+
+export const MAXIMUM_ENTRIES_PER_PAGE = 100;
```

Comment:
This is not used anywhere? Also a bit of an ambiguous name.

Reply:
Ah yeaup. was part of an earlier approach. thanks for flagging